src/components/Modal/ModalFooter/ModalFooterButtons/ModalFooterButtons.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { forwardRef } from "react";
+import VibeComponentProps from "../../../../types/VibeComponentProps";
+import VibeComponent from "../../../../types/VibeComponent";
+import { getTestId } from "../../../../tests/test-ids-utils";
+import { ComponentDefaultTestId } from "../../../../tests/constants";
+import { NOOP } from "../../../../utils/function-utils";
+import Flex from "../../../Flex/Flex";
+import Button from "../../../Button/Button";
+import ModalFooter from "../ModalFooter";
+
+export interface ModalFooterButtonsProps extends VibeComponentProps {
+  primaryButtonText?: string;
+  secondaryButtonText?: string;
+  onPrimaryButtonClick?: () => void;
+  onSecondaryButtonClick?: () => void;
+}
+
+const ModalFooterButtons: VibeComponent<ModalFooterButtonsProps> = forwardRef(
+  (
+    {
+      primaryButtonText = "Confirm",
+      secondaryButtonText = "Cancel",
+      onPrimaryButtonClick = NOOP,
+      onSecondaryButtonClick = NOOP,
```

Comment:
I assume it will work also without assigning a default NOOP action

Reply:
I can just go with () => {} instead, would you prefer it instead? or to give up entirely on a default?
If we are giving up on a default entirely we'll need to add a check before calling the function in the `onClick`.

Btw, I know it can work without the NOOP, but I wanted to make sure I follow existing practices

Reply:
@shlomitc pushed fixes for the other comments, lmk what option you prefer here

Reply:
the button component should not how to manage it already, so you can just remove the defaults completely